addons/mass_mailing/tests/test_mailing_list.py

```diff
@@ -36,6 +36,8 @@ def test_mailing_contact_to_list(self):
         wizard = wizard_form.save()
         action = wizard.action_add_contacts()
         self.assertEqual(contacts.list_ids, mailing)
+        create_dates = contacts.subscription_ids.mapped('create_date')
+        self.assertTrue(all(create_dates), "Some subscriptions have no Subscription Date.")
```

Comment:
I'm a big proponent of "Should", "Should not" in assertion messages. Say what/why we expect something, not what we have.
"All subscriptions should have a create_date"